prow-configurations/config.yaml.tpl

```diff
@@ -12,6 +24,16 @@ presubmits: # runs on PRs
       containers:
       - image: alpine
         command: ["/bin/printenv"]
+  - name: kyma-integration
```

Comment:
In my previous review, I mentioned that we should probably set MAX_CONCURRENCY. What do you think?